src/commcare_cloud/ansible/group_vars/redis.yml

```diff
@@ -6,7 +6,9 @@ redis_local_facts: false
 redis_logfile: "{{ redis_dir }}/redis.log"
 redis_service_name: redis-server
 redis_syslog_enabled: "yes"
-redis_maxmemory: 4gb
+# This implements the best practice of setting redis_maxmemory to half the RAM
+# Discussion: https://github.com/dimagi/commcare-cloud/pull/2929#issuecomment-499970466
+redis_maxmemory: "{{ ansible_memory_mb.real.total // 2 }}m"
```

Comment:
Can you add 4gb to the environment files that didn't have this defined before? I'm mostly concerned about swiss/pna which have 32 GB of memory, but we don't want redis to use 16 GB

Reply:
Yes that's a very good point. Thanks